examples/slack-clone/nextjs-slack-clone-dotenvx/README.md

```diff
@@ -0,0 +1,192 @@
+# Integrating Dotenvx with a Supabase Project
+
+This project is a full-stack Slack clone built using:
+
+### Frontend:
+
+- **[Next.js](https://github.com/vercel/next.js):** A React framework optimized for production.
+- **[Supabase.js](https://supabase.com/docs/library/getting-started):** For user management and real-time data syncing.
+
+### Backend:
+
+- **[Supabase](https://supabase.com/dashboard):** A hosted Postgres database with a RESTful API, used alongside Supabase.js.
+- **GitHub Authentication:** For user login.
+
+---
+
+## Introduction
+
+This example demonstrates how to use [dotenvx](https://dotenvx.com/) and `config.toml` to manage multiple environments seamlessly. You'll learn how to set up local and production environments with shared, secure configurations.
+
+---
+
+## Core Concept
+
+`config.toml` supports environment variables through the `env()` syntax. Using `dotenvx`, you can securely manage sensitive values like GitHub credentials for third-party authentication.
+
+### Key Features of Dotenvx:
+
+- Secrets are encrypted and stored securely, while private decryption keys are saved in `.env.keys` (excluded from version control).
+- Teams can share private keys to decrypt environment values securely.
```

Comment:
```suggestion
- Teams can share public keys to encrypt environment values securely.
- Private keys should be kept secret and only accessible through CI/CD environments.
```

Let's not recommend sharing private key in case it's misused.